src/pages/courses/index.vue

```diff
@@ -5,7 +5,7 @@
 		</section>
 		<section class="section">
 			<base-button
-				class="button is-info"
+				class="button is-primary"
```

Comment:
Evtl 
```suggestion
				design="primary"
```
sobald #200 durch ist.